### PR TITLE
Fix/sprint2 artem

### PR DESCRIPTION
**Pull request by RayMet**

### CODE REVIEW

File: client/src/components/ChangePassword.jsx [hunk 1]
```diff
@@ -50,11 +51,14 @@ export class ChangePassword extends Component {
                     <Input
                       id="check_new_password"
                       value={ this.state.check_new_password}
+                      size={40}
                       onChange={ this.onChangeHandler }
                       placeholder="Confirm Password"
                       type="password"
                       label="confirm Password"
```
Comment (Merteg) [inline client/src/components/ChangePassword.jsx]: I think "Confirm Password" or "Confirm password" seems prettier.

File: client/src/components/ChangePassword.jsx [hunk 2]
```diff
@@ -50,11 +51,14 @@ export class ChangePassword extends Component {
                     <Input
                       id="check_new_password"
                       value={ this.state.check_new_password}
+                      size={40}
```
Comment (Merteg) [inline client/src/components/ChangePassword.jsx]: Why do you restrict user by 40 symbols?

File: client/src/components/Nav/index.jsx [hunk 3]
```diff
@@ -33,10 +34,25 @@ class Nav extends Component {
                  </React.Fragment>)
   }
 
+  renderNavRecover() {
+    return (
+      <React.Fragment>
+        {
+        <NavItem><Link to={"/recover"}>forgot password?</Link>
+        </NavItem>
```
Comment (Merteg) [inline client/src/components/Nav/index.jsx]: When you add a link like this, it raises warning `<a> cannot appear as a descendant of <a>`. Better to do it this way:
`<li><Link to={"/recover"}>Forgot password?</Link></li>`

File: client/src/components/Nav/index.jsx [hunk 4]
```diff
@@ -33,10 +34,25 @@ class Nav extends Component {
...
+        }
+      </React.Fragment>
+    );
+  }
+
+
   render() {
     return (
-      <Navbar brand="Event.me" right>
+      <Navbar brand="Event.me " className=""  right>
```
Comment (Merteg) [inline client/src/components/Nav/index.jsx]: Remove empty className

File: client/src/components/RecoverPassword.jsx [hunk 5]
```diff
@@ -19,31 +19,38 @@ export class RecoverPassword extends Component {
...
                       placeholder="Email"
                       type="email"
                       label="Email"
                     />
-                    <Button waves='light' onClick={ this.sendMailRecoverPassword }>Send</Button>
+                </Col>
+                <Col className="center-align" offset="s7" s={4}>
+            <Button waves='light'
+            onClick={ this.sendMailRecoverPassword } >
+            Reset password</Button>
```
Comment (Merteg) [inline client/src/components/RecoverPassword.jsx]: Wrong indentation.

**Review by Merteg — CHANGES_REQUESTED** [review]

File: client/src/components/ChangePassword.jsx (hunk 1 above)
Comment (RayMet) [reply client/src/components/ChangePassword.jsx]: gitlab have Confirm Password

File: client/src/components/Nav/index.jsx (hunk 3 above)
Comment (RayMet) [reply client/src/components/Nav/index.jsx]: change NavItem on li?

Comment (Merteg) [reply client/src/components/Nav/index.jsx]: Yes

File: client/src/components/ChangePassword.jsx (hunk 2 above)
Comment (RayMet) [reply client/src/components/ChangePassword.jsx]: thought that made state size off input

Comment (Merteg) [reply client/src/components/ChangePassword.jsx]: This input is limited by col size, so size attribute doesn't impact on my screen. (But on the screen with a high resolution this input can seem broken)

File: client/src/components/ChangePassword.jsx (hunk 1 above)
Comment (Merteg) [reply client/src/components/ChangePassword.jsx]: But you have 'confirm Password'

**Comment by RayMet** [comment]
update

**Comment by RayMet** [comment]
New update

File: client/src/components/Nav/index.jsx [hunk 6]
```diff
@@ -33,10 +34,25 @@ class Nav extends Component {
                  </React.Fragment>)
   }
 
+  renderNavRecover() {
+    return (
+      <React.Fragment>
+        {
+        <li><Link to={"/recover"}>forgot password?</Link>
+        </li>
```
Comment (Merteg) [inline client/src/components/Nav/index.jsx]: Please use pretty indent for embedded tags. [styleCode](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-wrap-multilines.md)

File: client/src/components/Nav/index.jsx [hunk 7]
```diff
@@ -33,10 +34,25 @@ class Nav extends Component {
...
+        }
+      </React.Fragment>
+    );
+  }
+
+
   render() {
     return (
-      <Navbar brand="Event.me" right>
+      <Navbar brand="Event.me " right>
```
Comment (Merteg) [inline client/src/components/Nav/index.jsx]: Delete this space after `Event.me`

**Comment by RayMet** [comment]
Fixed

**Review by Merteg — APPROVED** [review]

**Review by yazhyvyi — COMMENTED** [review]
Tested the code, works fine for me. At this point, any changes that you want to add, change, better discuss with me

**Review by yazhyvyi — APPROVED** [review]

**Comment by yazhyvyi** [comment]
You must resolve all incoming conflicts before you merge your changes

**Pull request merged**